CHANGELOG.md

```
@@ -1,18 +1,42 @@
+- Github improvements/fixes (@CoMPaTech)
+- Async fixes (@ArnoutD - #141)
+
+## v0.40.0 (a22)
+
+- Correcting messageflow to HA
+
+## v0.40.0 (a4)
+
+Full rewrite of library into async version (@brefra).
+Main list of changes:
+- Full async and typed
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)